website/docs/r/kusto_cluster.html.markdown

```diff
@@ -82,7 +82,9 @@ The following arguments are supported:
 
 * `zones` - (Optional) Specifies a list of Availability Zones in which this Kusto Cluster should be located. Changing this forces a new Kusto Cluster to be created.
 
-* `engine` - (Optional). The engine type that should be used. Possible values are `V2` and `V3`. Defaults to `V2`.
+* `engine` - (Optional). The engine type that should be used. Possible values are `V2` and `V3`. Defaults to `V2`. Changing this forces a new Kusto Cluster to be created.
+
+~> **NOTE:** In v4.0 and later version of the AzureRM Provider, default engine type will be changed to `V3`. 
```

Comment:
quotes
```suggestion
~> **NOTE:** In `v4.0.0` and later version of the AzureRM Provider, default engine type will be changed to `V3`. 
```

and we should be making this change in code behind the 4.0 flag so its not missed and can be trial'd?

Reply:
> 

https://github.com/hashicorp/terraform-provider-azurerm/blob/main/internal/services/kusto/kusto_cluster_resource.go#L294-L301 the code we need has been added, only the doc needs to be updated.

Reply:
ahh perfect! thanks